docs/migration-from-hermione.md

```diff
@@ -0,0 +1,23 @@
+## Migration from Hermione
+
+Testplane@1 is backward compatible with Hermione@8.
+
+So, to migrate from Hermione to Testplane you need to:
+- Uninstall hermione (`npm uninstall hermione`);
+- Install testplane (`npm install -D testplane`);
+- Replace all `import ... from "hermione"` and `require("hermione")` to `import ... from "testplane"` and `require("testplane")`;
+- Replace `declare module "hermione"` to `declare module "testplane"`;
+- Replace "hermione" to "testplane" in `compilerOptions.types` of your `tsconfig.json` files.
+
+Other than that, everything should be fine. Hermione plugins are fully compatible with Testplane@1.
+
+These things only work in Testplane@1 and will be removed in Testplane@2:
```

Comment:
I'm not sure we need to talk about it right now

Reply:
I wrote it to show "you dont need to change any of these in Testplane@1"